File: 백준에서 가장 많이 출린 문제 TOP 100/2438.c
//나만의 방식
#include <stdio.h>
int main() {
	int input;

	scanf("%d", &input);
	int star = input;

	star = star - star + 1;
	for (int i = 0; i < input; i++) {
		for (int j = 0; j < star; j++) {
			printf("*");
		}
		printf("\n");
		star++;
	}
}

//정답
#include <stdio.h>

int main() {
	int n;
	scanf("%d", &n);
	for (int i = 1; i <= n; ++i) {
		for (int j = 1; j <= i; ++j) {
			printf("*");
		}
		puts("");
	}
	return 0;
}

File: Python/2438.py
# 시도 1
n = int(input())
for i in range(n):
    for _ in range(i+1):
        print('*',end='')
    print()


# 시도 2
n = int(input())
for i in range(1,n+1):
    for j in range(i):
        print('*',end='')
    print()


# 인터넷 답지
n=int(input())
for i in range(1,n+1):
    print("*"*i) # * 를 i의 값만큼 곱해서 출력